### PR TITLE
[Feature] : Provide destDir as an option.

### PR DESCRIPTION
**Pull request by SparshithNR**
1. This will avoid having a stew.mv followed by mergeTree.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -26,6 +27,8 @@ BroccoliMergeTrees.prototype.build = function() {
   if (this.mergeTrees == null) {
     // Defer instantiation until the first build because we only
     // have this.inputPaths and this.outputPath once we build.
+    this.outputPath = this.options.destDir ? this.outputPath + '/' + this.options.destDir : this.outputPath;
+    fs.ensureDirSync(this.outputPath);
```
Comment (stefanpenner) [inline index.js]: Can we avoid the extra IO operation if not needed? (it wasn't not always needed before)

**Comment by stefanpenner** [comment]
@SparshithNR was this needed for that performance work we did earlier? Did we forget about it?

File: index.js (hunk 1 above)
Comment (SparshithNR) [reply index.js]: Fixed this.

**Comment by SparshithNR** [comment]
Somehow we missed this. Now I have modified code to allow the `destDir` option in this change.

**Comment by stefanpenner** [comment]
released as v4.1.0 🎉

**Pull request merged**